releaseNotes/AzureDevOpsPushPipelineExtension.md

```diff
@@ -3,7 +3,43 @@ Azure App Configuration Push extension for Azure DevOps pipeline can be installe
 ![sample](pictures/AzureDevOpsPushExtensionVersionSample.PNG)
 
 ### v5.0.0 - February, 02 2023
-Updated the task to require Node.js 16. It previously required 10
+**Breaking Changes**
+  - Updated the task to require Node.js 16. It previously required 10.
+  - Updated the minimum supported azure pipeline agent version to [2.206.1](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v2.206.1) or later. Previously it was [2.144.0](https://github.com/microsoft/azure-pipelines-agent/releases/tag/v2.144.0).
+  - Importing configurations with values as escaped JSON strings and the content type `application/json`, introduces a modified behavior, the values of properties in the settings are expected to be specified as JSON objects. Please ensure the values are JSON objects.
+
+    **Before**
+    ```
+    {  
+      "app": "{\"uri\":\"https://keyvault.vault.azure.net/secrets/secret\"}"
+    }
+
+    task: AzureAppConfigurationPush@4
+    inputs:
+      azureSubscription: '<subscription>'
+      AppConfigurationEndpoint: '<store endpoint>'
+      ....
+      ContentType: 'application/vnd.microsoft.appconfig.keyvaultref+json;charset=utf-8'
+    ```
+
+    **After**
+    ```
+    {  
+      "app": {
+          "uri": "https://keyvault.vault.azure.net/secrets/secret " 
+        }
+    }
+
+    task: AzureAppConfigurationPush@5
+    inputs:
+      azureSubscription: '<subscription>'
+      AppConfigurationEndpoint: '<store endpoint>'
+      ....
+      ContentType: 'application/vnd.microsoft.appconfig.keyvaultref+json;charset=utf-8'
+
+    ```
+
+  - Feature flag import now requires adherence to the [feature management schema](https://github.com/microsoft/FeatureManagement-Dotnet/blob/release/v3/docs/schemas/FeatureManagement.v1.0.0.json). Settings with feature flag prefix and content type will fail to import

```

Comment:
Does specifying the feature flags as the [feature management schema](https://github.com/microsoft/FeatureManagement-Dotnet/blob/release/v3/docs/schemas/FeatureManagement.v1.0.0.json) for import then get converted to the App Config service prefix and content type for feature flags by the extension or the app config service?  So that the imported feature flags appear in the Feature Manager view alongside feature flags manually created in the portal?